File: patch_sheet_3.js
const fs = require('fs');
const file = '/Users/ojaeeul/Downloads/세종요리제과학원/무제 폴더/수정전/sejk 4/sejongcook-next/Sejong/SejongAttendance/public/sheet.html';
let content = fs.readFileSync(file, 'utf8');

let search = `function calculateGlobalCarryOver(memberId, targetYear, targetMonth, targetDay, cutoffOverrides = {}, attendanceCutoffs = {}, courseFilter = null) {
            const mIdStr = String(memberId);`;

let replace = `function calculateGlobalCarryOver(memberId, targetYear, targetMonth, targetDay, cutoffOverrides = {}, attendanceCutoffs = {}, courseFilter = null) {
            const mIdStr = String(memberId);
            const m = window.membersData ? window.membersData.find(x => String(x.id) === mIdStr) : undefined;`;

content = content.replace(search, replace);
fs.writeFileSync(file, content);

File: patch_sheet3.py
with open('/Users/ojaeeul/Downloads/세종요리제과학원/무제 폴더/수정전/sejk 4/sejongcook-next/Sejong/SejongAttendance/public/sheet.html', 'r', encoding='utf-8') as f:
    content = f.read()

new_eraser = """                    setCellSymbol(td, 'unchecked');
                    await saveAttendance(memberId, dateStr, 'unchecked', 'ALL');"""
content = content.replace("                    await saveAttendance(memberId, dateStr, 'unchecked', 'ALL');", new_eraser)

with open('/Users/ojaeeul/Downloads/세종요리제과학원/무제 폴더/수정전/sejk 4/sejongcook-next/Sejong/SejongAttendance/public/sheet.html', 'w', encoding='utf-8') as f:
    f.write(content)
